manifests/0000_90_ingress-operator_03_prometheusrules.yaml

```diff
@@ -64,3 +64,24 @@ spec:
         record: cluster:route_metrics_controller_routes_per_shard:median
       - expr: sum (openshift_route_info) by (tls_termination)
         record: cluster:openshift_route_info:tls_termination:sum
+    - name: openshift-ingress-to-route-controller.rules
+      rules:
+        - alert: IngressWithoutClassName
+          expr: openshift_ingress_to_route_controller_ingress_without_class_name == 1
+          for: 1d
+          labels:
+            severity: warning
+          annotations:
+            summary: Ingress without IngressClassName for 1 day
+            description: "This alert fires when there is an Ingress with an unset IngressClassName for longer than one day."
+            message: "Ingress {{ $labels.name }} is missing the IngressClassName for 1 day."
+        - alert: UnmanagedRoutes
+          expr: openshift_ingress_to_route_controller_route_with_unmanaged_owner == 1
+          for: 5m
```

Comment:
Are you sure we want to see this alert every 5m?

Reply:
This is actually the time between a case being evaluated as `Pending` and as `Firing`. Prometheus will wait for 5m, and if the route still has an unmanaged owner, then it will start firing the alert. But, you're right, 5m could be too less for this case. How about 30m or would you have another suggestion?

Reply:
I'm not 100% sure what problems it causes. If it causes no problem to be in that state, then once a day should be fine.  If it causes problems to be in that state, then maybe once an hour?

Reply:
The time is not specified for this metric on the BZ. For the other metric, it was specified as once a day. I changed this one to 1h, and we can update it later on if this is too frequent.